tests/Doctrine/Tests/ORM/Proxy/ProxyFactoryTest.php

```diff
@@ -139,6 +140,29 @@ public function testFailedProxyLoadingDoesNotMarkTheProxyAsInitialized(): void
         $this->assertInstanceOf('Closure', $proxy->__getCloner(), 'The cloner wasn\'t removed');
     }
 
+    public function testExceptionWhenLoadingProxyDoesNotMarkTheProxyAsInitialized(): void
+    {
+        $persister = $this->getMockBuilder(BasicEntityPersister::class)->setMethods(['load'])->disableOriginalConstructor()->getMock();
```

Comment:
```suggestion
        $persister = $this->getMockBuilder(BasicEntityPersister::class)
            ->setMethods(['load'])
            ->disableOriginalConstructor()
            ->getMock();
```

Reply:
Formatting is identical to the one above. I guess if the Doctrine projects wants a different way of formatting fluent interfaces, introducing a CS rule and applying it across the whole project would make more sense. What do you think?

Reply:
> I guess if the Doctrine projects wants a different way of formatting fluent interfaces

This is not specifically about fluent interfaces: this line is 135 characters long, while PSR recommends 120.

> introducing a CS rule and applying it across the whole project would make more sense. What do you think?

It would make sense, but although it sounds simple, it sure isn't, see http://journal.stuffwithstuff.com/2015/09/08/the-hardest-program-ive-ever-written/